website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -33,6 +33,7 @@ the traffic redirection command is automatically set up via an init container.
 * To use transparent proxy on Kubernetes, Consul-helm >= `0.32.0` and Consul-k8s >= `0.26.0` are required in addition to Consul >= `1.10.0`.
 * If the default policy for ACLs is "deny", then Service Intentions should be set up to allow intended services to connect to each other.
 Otherwise, all Connect services can talk to all other services.
+* Linux worker nodes within a Kubernetes cluster should have `iptables` installed. 
```

Comment:
🤔 I don't think we need worker nodes to have iptables - we only need containers images to have it installed (and they do). As an aside, I think all k8s nodes should have iptables since that's how kube-proxy does the routing.

Reply:
I'll look into it a bit more but will not hold the approval.

Reply:
Yeah good point. Unless they're using nftables instead maybe? I think the underlying problem is that iptable command in the container needs hooks to the kernel module, which is what must be loaded on the host, unless I'm mistaken?